### ISSUE TITLE
Block System

### ISSUE THREAD

**Issue by aggelosQQ** [TotalFreedom/TotalFreedomMod#781]
Can we have a TF command like /block or /ignore? If I use it to somebody his messages and stuff won't show up on my chat.

This will be mostly for OPs, not admins as admins should see everything when OPs doesn't have to.
OPs won't be able to use it to admins though so they won't ignore their warning.

**Comment by tylerhyperHD** [comment]
I have recently made a pull request that does that in Essentials and it makes it so that only admins are exempted instead of ops.
https://github.com/TotalFreedom/TF-Essentials/pull/6

**Comment by Wild1145** [comment]
This has already been requested and I believe rejected...

**Comment by tylerhyperHD** [comment]
I get so many people in action that I just do not want to message me, and I think that it should be something that shouldn't be rejected.

**Comment by JeromSar** [comment]
I don't think we want this implemented. All players should be able to see everyone's messages in my opinion. What do you think @Wild1145 ?

**Comment by aggelosQQ** [comment]
As I said, this is mostly for OPs that don't want unwanted stuff such as advertising when no admins are online, it would make their lifes easier without anybody bothering them

**Comment by Wild1145** [comment]
@JeromSar I see no problem with Op's blocking other Op's messages, but they shouldnt be able to ignore admins. Maybe have it so you can not block anyone of a higher rank than you?

**Comment by tylerhyperHD** [comment]
That's exactly what I did in my essentials pull request.

**Comment by JeromSar** [comment]
@Wild1145 Good idea
Accepted.

**Comment by Wild1145** [comment]
Accepted providing it is only possible for Op's to block Op's and admins to block Op's (Nobody should be able to block an admin)

**Comment by JeromSar** [comment]
I would like to add that admins should not be able to block anyone. Admins should always see the full chat in my opinion.
